package.json

```diff
@@ -25,6 +25,7 @@
     "csv-parse": "^1.1.7",
     "express": "^4.14.0",
     "node-postal": "^0.3.0",
+    "pbf2json": "^4.2.0",
```

Comment:
the dockerfile needs to have a couple lines removed if we are using the binaries from `npm`:
https://github.com/pelias/interpolation/blob/master/Dockerfile#L49-L51